### PR TITLE
Explorer delegations

### LINKED ISSUE #4221: Explorer - Delegate to mixnodes

### PR DESCRIPTION
**Pull request by fmtabbara**
# DO NOT MERGE

# Description

Allow delegations through the explorer using the Keplr wallet

Closes: #4221 

# Checklist:

- [ ] added a changelog entry to `CHANGELOG.md`

### CODE REVIEW

File: explorer/src/components/ConnectKeplrWallet.tsx
Comment (fmtabbara) [inline explorer/src/components/ConnectKeplrWallet.tsx:1]: :heavy_plus_sign:

File: explorer/src/components/Delegations/ConfirmationModal.tsx
Comment (fmtabbara) [inline explorer/src/components/Delegations/ConfirmationModal.tsx:1]: ➕

File: explorer/src/components/Delegations/DelegateIconButton.tsx
Comment (fmtabbara) [inline explorer/src/components/Delegations/DelegateIconButton.tsx:1]: ➕

File: explorer/src/components/Delegations/DelegateModal.tsx
Comment (fmtabbara) [inline explorer/src/components/Delegations/DelegateModal.tsx:1]: ➕

File: explorer/src/components/Delegations/DelegationModal.tsx
Comment (fmtabbara) [inline explorer/src/components/Delegations/DelegationModal.tsx:1]: ➕

File: explorer/src/components/Delegations/ErrorModal.tsx
Comment (fmtabbara) [inline explorer/src/components/Delegations/ErrorModal.tsx:1]: ➕

File: explorer/src/components/Delegations/LoadingModal.tsx
Comment (fmtabbara) [inline explorer/src/components/Delegations/LoadingModal.tsx:1]: ➕

File: explorer/src/components/Delegations/ModalListItem.tsx
Comment (fmtabbara) [inline explorer/src/components/Delegations/ModalListItem.tsx:1]: ➕

File: explorer/src/components/Delegations/SimpleModal.tsx
Comment (fmtabbara) [inline explorer/src/components/Delegations/SimpleModal.tsx:1]: ➕

File: explorer/src/context/cosmos-kit.tsx
Comment (fmtabbara) [inline explorer/src/context/cosmos-kit.tsx:1]: ➕

File: explorer/src/index.tsx
Comment (fmtabbara) [inline explorer/src/index.tsx:1]: ➕

**Review by gala1234 — APPROVED** [review]

**Comment by fmtabbara** [comment]
replaced by https://github.com/nymtech/nym/pull/4470

**Pull request closed without merging**